src/module/sheets/TwodsixActorSheet.ts

```diff
@@ -61,26 +59,8 @@ export class TwodsixActorSheet extends AbstractTwodsixActorSheet {
 
     html.find('.roll-damage').on('click', (this._onRollDamage.bind(this)));
 
-    html.find('.stat-damage').on('change', this._setDamageFromEvent.bind(this));
-    html.find('.special-damage').on('change', this._setDamageFromEvent.bind(this));
   }
 
-  private async _setDamageFromEvent(event:Event):Promise<void> {
```

Comment:
This seems to not be needed? Or am I missing something?

Reply:
Yeah, this is dead code.